### ISSUE TITLE
Build error on linux

### ISSUE THREAD

**Issue by jpfairbanks** [JuliaIO/ImageMagick.jl#46]
Hello,

I cannot install this package on ubuntu 16.04, even though libmagickwant6.q16-2 is installed from apt.
From the julia v0.4.5 prompt:

```
julia> Pkg.build("ImageMagick")
INFO: Building Rmath
INFO: Building ImageMagick
Installing dependency libmagickwand-6.q16-2 via `sudo apt-get install libmagickwand-6.q16-2`:
sudo: no tty present and no askpass program specified
========================[ ERROR: ImageMagick ]=============================

LoadError: failed process: Process(`sudo apt-get install libmagickwand-6.q16-2`, ProcessExited(1)) [1]
while loading /home/james/.julia/v0.4/ImageMagick/deps/build.jl, in expression starting on line 98

================================================================

============================[ BUILD ERRORS ]=====================

WARNING: ImageMagick had build errors.

 - packages with build errors remain installed in /home/james/.julia/v0.4
 - build the package(s) and all dependencies with `Pkg.build("ImageMagick")`
 - build a single package by running its `deps/build.jl` script

================================================================
```

running `sudo apt-get install libmagickwand-6.q16-2` shows that the library is already installed.

Shouldn't apt-get install libmagickwand... be unnecessary because the package is already installed?
Should an askpass program be provided?

I don't know enough about the Bindeps system to fix it.

**Comment by timholy** [comment]
It must not be finding it. `deps/build.jl` lists a number of paths that are scanned for your library, you can try adding to the names there.

**Comment by nalimilan** [comment]
The sudo error is https://github.com/JuliaLang/BinDeps.jl/issues/199. But the fact that it keeps trying to reinstall the package is an ImageMagick.jl bug.

**Comment by musm** [comment]
try now?

**Comment by SimonDanisch** [comment]
Should be fixed by #121! Please try again on master!
